src/containers/settings/Settings.tsx

```diff
@@ -16,13 +20,35 @@ const Settings: React.FC = () => {
       </Helmet>
       <div className="content-container">
         <Title>Settings</Title>
-        <Form name="basic" onFinish={onFinish}>
+        <Form name="basic">
           <Form.Item>
             <Button type="primary" htmlType="submit">
               Delete Account
             </Button>
           </Form.Item>
         </Form>
+
+        <Form name="basic" onFinish={onFinishChangePassword}>
+          <Form.Item
+            label="Current Password"
+            name="currentPassword"
+            rules={[{ required: true, message: 'Please input your password!' }]}
+          >
+            <Input.Password />
+          </Form.Item>
+          <Form.Item
+            label="New Password"
+            name="newPassword"
+            rules={[{ required: true, message: 'Please input your password!' }]}
+          >
+            <Input.Password />
+          </Form.Item>
```

Comment:
Add a retype field for the new password? Or give the user a way to see what they typed?